src/iterative_sorting/iterative_sorting.py

```diff
@@ -3,19 +3,13 @@ def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr)):
         smallest_index = i
-        for j in range(i + 1, len(arr) - 1):
+        for j in range(i + 1, len(arr)):
             if arr[j] < arr[smallest_index]:
                 smallest_index = j
-        temp = arr[smallest_index]
-        arr[smallest_index] = arr[i]
-        arr[i] = temp
-        # for j in range(i + 1, len(arr) - 1):
-        #     if arr[j] < arr[smallest_index]:
-        #         smallest_index = j
-
-        # temp = arr[smallest_index]
-        # arr[smallest_index] = arr[cur_index]
-        # arr[cur_index] = temp
+        temp = arr[i]
+        arr[i] = arr[smallest_index]
+        arr[smallest_index] = temp
+
```

Comment:
you can also do a one liner which i think looks cleaner in python: a,b=b,a or arr[i],arr[small]=arr[small],arr[i]